blog-post.html

```diff
@@ -2,11 +2,92 @@
 <html>
   <head>
     <title>Blog Post</title>
+    <style>
+      @import "index.css";
+    </style>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
   </head>
   <body>
-    <section id="header"></section>
-    <section id="main"></section>
+
+    <section id="header">
+      <h1><a class="header-link" href="index.html">POST ONLY</a></h1>
+      <H6>A PAGE WITH JUST A BLOG POST CONTENT.</H6>
+    </section>
+<br>
+<br>
+    <section id="main4">
+      <div class="main-content4-wrapper">
+        <div class="main-content4">
+          <h4 class="main-content4-text1">THIS IS A <b>BLOG POST </b>ONLY LAYOUT</h4>
+        </div>
+        <div class="main-content4-img-container">
+          <img class="main-content4-img" src="https://picsum.photos/300/205"/>

```

Comment:
Try to use better variables such as "blogPostMain" or something rather then just "main4" or other variations of that